Assets/Scripts/GameService.cs

```diff
@@ -72,7 +76,35 @@ private void Update()
 
     public SoundService GetSoundService() => soundService;
 
-    public UIView GetUIService() => uiService; 
+    public UIView GetUIService() => uiService;
+
+    public MainMenu GetMainMenu() => mainMenu;
     #endregion
 
+    public void SetDifficultyVariables()
```

Comment:
This is not the correct location in code for this method. Ideally if many components are getting affected by the difficulty level in the game, we should be having a separate DifficultyService which contains all the difficulty level logic.